selftests/unit/test_utils_process.py

```diff
@@ -218,6 +252,16 @@ def test_run_sudo_shell_uid_0(self):
         p = process.run(cmd='ls -l', sudo=True, shell=True, ignore_status=True)
         self.assertEqual(p.command, expected_command)
 
+    @unittest.skipUnless(ECHO_CMD, "Echo command not available in system")
+    def test_run_unicode_output(self):
```

Comment:
Well, this test doesn't work for python2. There, the `stdout_text` is 'Avok\xc3\xa1do\n' because `stdout` is type `str.`

Reply:
@apahim what do you get when running:

```python
from avocado.utils.process import run

result = run("/usr/bin/echo %s" % u"Avok\xe1do", encoding="utf-8", env={"LC_ALL":"en_US.UTF-8"})
print(result.stdout_text)
print(map(ord, result.stdout_text))
```

Reply:
```
~/git/avocado (clebergnu-avocado_python2_utf8-test)$ python
Python 2.7.14 (default, Mar 14 2018, 13:36:31) 
[GCC 7.3.1 20180303 (Red Hat 7.3.1-5)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from avocado.utils.process import run
>>> 
>>> result = run("/usr/bin/echo %s" % u"Avok\xe1do", encoding="utf-8", env={"LC_ALL":"en_US.UTF-8"})
>>> print(result.stdout_text)
Avokádo

>>> print(map(ord, result.stdout_text))
[65, 118, 111, 107, 195, 161, 100, 111, 10]
>>> 

>>> 

```

Reply:
Ok, it turns out to be an issue with my local user. Merging.